src/main/java/com/softvision/library/tdd/model/Book.java

```diff
@@ -0,0 +1,68 @@
+package com.softvision.library.tdd.model;
+
+import com.fasterxml.jackson.annotation.*;
+import org.hibernate.annotations.OnDelete;
+import org.hibernate.annotations.OnDeleteAction;
+import org.springframework.data.jpa.domain.support.AuditingEntityListener;
+
+import javax.persistence.*;
+
+@Entity
+@EntityListeners(AuditingEntityListener.class)
+public class Book extends BaseEntity{
+	
+	@Id
```

Comment:
You can set this the strategy here and in Author as IDENTITY, such that id of books can also start at 1, regardless if created multiple authors before adding first book to author 1.

@Id
@GeneratedValue(strategy = GenerationType.IDENTITY)